docs/real-time-analytics/mainframe-offload/process.sql

```diff
@@ -0,0 +1,37 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create stream of transactions from the Kafka topic
+CREATE STREAM mq_transactions (
+  dep_account_no STRING,
+  dep_balance_dollars BIGINT,
+  dep_balance_cents BIGINT,
+  timestamp BIGINT
+) WITH (
+  KAFKA_TOPIC = 'mq_transactions',
+  VALUE_FORMAT = 'JSON',
+  PARTITIONS = 6
+);
+
+-- Normalize the data and calculate timestamp deltas
+CREATE STREAM mq_transactions_normalized WITH (KAFKA_TOPIC = 'mq_transactions_normalized')
+  AS SELECT
+    dep_account_no,
+    CAST(CONCAT(CAST(dep_balance_dollars AS STRING),'.',CAST(dep_balance_cents AS STRING)) AS DOUBLE) AS BALANCE,
```

Comment:
Should `BALANCE` also be lower case to match the other fields of the query?

Reply:
@rspurgeon Yes, please see latest commit, that also has updated logic (thanks to you for identifying & helping to resolve the issue!)